src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -265,6 +292,11 @@ type MessageDbContext(connection : MessageDbConnection, batchOptions : BatchOpti
     member _.LoadBatched streamName log tryDecode : Async<StreamToken * 'event array> = async {
         let! version, events = Read.loadForwardsFrom log connection.ReadRetryPolicy connection.ReadConnection batchOptions.BatchSize batchOptions.MaxBatches streamName 0L
         return Token.create version, Array.chooseV tryDecode events }
+    member _.LoadLast streamName log tryDecode : Async<StreamToken * 'event array> = async {
+        let! version, event = Read.loadLastEvent log connection.ReadRetryPolicy connection.ReadConnection streamName
+        let toArray = function | ValueSome e -> [| e |] | ValueNone -> [||]
```

Comment:
```suggestion
        let toArray = function ValueSome e -> [| e |] | ValueNone -> [||]
```
or make a `ValueOption.toArray` and put it in the infrastructure.fs if it does not exist

Reply:
Yeah, that's probably a better place to put it